also defined _6KZ, as clang -march=armv6kz/zk defines this, not _6KZ, as it's a deprecated name
https://reviews.llvm.org/D14568
--
diff --git a/Source/WTF/wtf/PlatformCPU.h b/Source/WTF/wtf/PlatformCPU.h
index e5c41a0..4f51ad4 100644
--- a/Source/WTF/wtf/PlatformCPU.h
+++ b/Source/WTF/wtf/PlatformCPU.h
@@ -162,6 +162,7 @@
     || defined(__ARM_ARCH_6J__) \
     || defined(__ARM_ARCH_6K__) \
     || defined(__ARM_ARCH_6Z__) \
+    || defined(__ARM_ARCH_6KZ__) \
     || defined(__ARM_ARCH_6ZK__) \
     || defined(__ARM_ARCH_6T2__) \
     || defined(__ARMV6__)
